### PR TITLE
New WORKING clones

### PR DESCRIPTION
**Pull request by clawgrip**
-----------------------------
Pac-Man (Cirsa, Spanish bootleg on Galaxian hardware) [Ricardo - Factory Arcade, Recreativas.org]
Komemokos (pacmanblci hack) [Ricardo - Factory Arcade, Recreativas.org]

### CODE REVIEW

File: src/mame/drivers/galaxian.cpp [hunk 1]
```diff
@@ -15061,6 +15106,8 @@ GAME( 1981, pacmanbl,    puckman,  pacmanbl,   pacmanbl,   galaxian_state, init_
...
 GAME( 1981, pacmanblb,   puckman,  pacmanbl,   pacmanblb,  galaxian_state, init_pacmanbl,   ROT90,  "bootleg",                      "Pac-Man (Moon Alien 'AL-10A1' hardware)",                    MACHINE_SUPPORTS_SAVE ) // Doesn't have separate tile / sprite roms, probably should move it
 GAME( 1981, pacmanblc,   puckman,  pacmanbl,   pacmanbl,   galaxian_state, init_pacmanbl,   ROT270, "bootleg (Calfesa)",            "Pac-Man (Calfesa, Spanish bootleg on Galaxian hardware)",    MACHINE_IMPERFECT_GRAPHICS | MACHINE_WRONG_COLORS | MACHINE_SUPPORTS_SAVE )  // same PROM as Moon Cresta, gives very strange colors and sprites get cut
+GAME( 1981, pacmanblci,  puckman,  pacmanbl,   pacmanbl,   galaxian_state, init_pacmanbl,   ROT270, "bootleg (Cirsa)",              "Pac-Man (Cirsa, Spanish bootleg on Galaxian hardware)",      MACHINE_SUPPORTS_SAVE )
+GAME( 199?, komemokos,   puckman,  pacmanbl,   pacmanbl,   galaxian_state, init_pacmanbl,   ROT270, "hack",                         "Komemokos (pacmanblci hack)",                                MACHINE_SUPPORTS_SAVE )
```
Comment (cuavas) [inline src/mame/drivers/galaxian.cpp:15110]: Please don’t put shortnames in descriptions.  They’re an implementation detail, and they get routinely changed.

File: src/mame/mame.lst [hunk 2]
```diff
@@ -14560,6 +14560,8 @@ pacmanbl                        // bootleg
 pacmanbla                       // bootleg
 pacmanblb                       // bootleg
 pacmanblc                       // bootleg (Calfesa)
+pacmanblci                      // bootleg (Cirsa)
+komemokos                       // hack
```
Comment (cuavas) [inline src/mame/mame.lst:14564]: Short names need to be sorted to keep these lists manageable.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/mame.lst (hunk 2 above)
Comment (clawgrip) [reply src/mame/mame.lst]: Ups! Fixed

File: src/mame/drivers/galaxian.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/drivers/galaxian.cpp:15110]: Ok, but it's the common approach on this driver. Should I change them all?

"Super Galaxians (galaxiana hack)"
"Galaxian Turbo (superg hack)"
"Space Invaders Galactica (galaxiana hack)"
"Galaxian Part X (moonaln hack)"
"Kamakazi III (superg hack)"
"Komekokos (pacmanblci hack)"

Comment (clawgrip) [reply src/mame/drivers/galaxian.cpp]: Ok, I've changed them all. Thx!

**Comment by Tafoid** [comment]
Checking up on this.. have the issues been addressed?

**Comment by cuavas** [comment]
`superg` is “Super Galaxians” – should further modification refer to the direct ancestor?

**Comment by clawgrip** [comment]
>  refer to the direct ancestor

Done, thanks

**Pull request merged**